OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile

```diff
@@ -0,0 +1,121 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2018, 2020 Oracle and/or its affiliates.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database 21c
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) db_home.zip
+#     Download Oracle Database 21c Enterprise Edition or Standard Edition 2 for Linux x64
+#     from http://www.oracle.com/technetwork/database/enterprise-edition/downloads/index.html
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Put all downloaded files in the same directory as this Dockerfile
+# Run: 
+#      $ docker build -t oracle/database:19.3.0-${EDITION} . 
+#
+# Pull base image
+# ---------------
+FROM oraclelinux:8-slim as base
```

Comment:
Please do not use `oraclelinux:8-slim`, as per the base image rules in the contribution guide: https://github.com/oracle/docker-images/blob/main/CONTRIBUTING.md#base-image-rules